index.html

```diff
@@ -219,5 +219,27 @@ <h2>Skills</h2>
         </ul>
       </div>
       </section> 
+
+      <section class="contact-section">
+        <div class="contact-head">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
+        <div class="form">
+            <input class="form-input" type="text" name="name" placeholder="Full name">
+            <input class="form-input" type="email" name="email" placeholder="Email address">
+            <textarea name="mail-message" id="" cols="20" rows="5" placeholder="Enter text here"></textarea>
+            <input class="send" type="submit" value="Get in touch">
+        </div>
```

Comment:
- [ ] As the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) suggest, kindly ensure that when the submit button is clicked the form inputs are collected. I suggest that you collect data by using the [Formspree service](https://formspree.io/html) as it has been clearly stated ☺️ .